src/Sulu/Bundle/RouteBundle/Controller/RouteController.php

```diff
@@ -53,18 +54,25 @@ class RouteController extends AbstractRestController implements ClassResourceInt
      */
     private $routeGenerator;
 
+    /**
+     * @var ConflictResolverInterface
+     */
+    private $conflictResolver;
+
     public function __construct(
         ViewHandlerInterface $viewHandler,
         RouteRepositoryInterface $routeRepository,
         EntityManagerInterface $entityManager,
         RouteGeneratorInterface $routeGenerator,
+        ConflictResolverInterface $conflictResolver,
```

Comment:
@alexander-schranz what is the correct way for doing this? can you give me a hint about an example somewhere? 🙂

Reply:
Fount one: https://github.com/sulu/sulu/blob/e13d40a362f27fa2fb4defbbff55404e9ccca707/src/Sulu/Bundle/MediaBundle/Controller/MediaController.php#L163-L168